addons/mrp/models/stock_move.py

```diff
@@ -322,6 +323,12 @@ def _onchange_quantity(self):
            float_compare(self.product_uom_qty, self.quantity, precision_rounding=self.product_uom.rounding) != 0:
             self.manual_consumption = True
 
+    @api.constrains('quantity')
+    def _check_negative_quantity(self):
+        for move in self:
+            if any(line.quantity < 0 for line in move.raw_material_production_id.move_raw_ids):
```

Comment:
```suggestion
            if move.raw_material_production_id and float_compare(move.quantity, 0, precision...) < 0):
```

Reply:
Hello :wave: 
Requested changes are done.
Thanks for the review :slightly_smiling_face: